lib/filter.g

```diff
@@ -209,18 +263,35 @@ end );
 ##  This means that after the above implication has been installed,
 ##  one can rely on the fact that every object in the filter
 ##  <C>IsGroup and IsCyclic</C> will also be in the filter
-##  <Ref Func="IsCommutative"/>.
+##  <Ref Func="IsCommutative"/>.<P/>
+##
+##  Adding logical implications can change the rank of filters 
+##  (see <Ref Func="RankFilter"/>) and consequently the rank, and so choice of
+##  methods for operations (see <Ref Sect="Applicable Methods and Method Selection"/>).
+##  By default <C>InstallTrueMethod</C> adjusts the method selection data structures 
+##  to take care of this, but this process can be time-consuming, so functions
+##  <Ref Func="SuspendMethodReordering"/> and <Ref Func="ResumeMethodReordering"/>
+##  are provided to allow control of this process.
```

Comment:
I think it's good to document this somewhere. But do we really want to document it for the public? I.e. perhaps move this out of the GAPDoc comment; or put it inside an XML comment?